site/content/events/2015-pittsburgh/contact/index.html

```diff
@@ -0,0 +1,17 @@
+---
+extension: html
+filter:
+- erb
+- markdown
+title: contact
+dirty: true
+layout: event_pitt
+---
+<% @eventhome = @page.directory.split(File::SEPARATOR)[0..1].join(File::SEPARATOR) %> 
+<% @eventid = File.basename(@page.directory) %>
+
+If you want to contact us by email at <a href='mailto:<%= render(:partial => "/#{@eventhome}/_email_organizers") %>'><%= render(:partial => "/#{@eventhome}/_email_organizers") %></a>
+
+** Our special team**
+
+<%= render(:partial => "/#{@eventhome}/_team_local") %>
```

Comment:
Please add this content to the bottom of this file:

https://github.com/jedi4ever/devopsdays-webby/blob/master/site/content/events/2015-template/contact/index.html#L19-L21